src/index.html

```diff
@@ -5,9 +5,50 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./styles/index.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles/index.scss">
   </head>
+
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <div class="card__image-container">
+        <img
+          src="/src/images/imac.jpeg"
+          alt="imac"
+          class="card__img"
+        >
+      </div>
+
+      <h1 class="card__product-name">
```

Comment:
it's better to use `h2` since it's not a page title
```suggestion
      <h2 class="card__product-name">
```